src/pages/accounts/NewPasswordForm.js

```diff
@@ -0,0 +1,128 @@
+    } else {
+      toast('Invalid fields on the form.');
+    }
+  };
+  return (
+    <>
+      <form onSubmit={onSubmit}>
+        <h3 className="text-2xl font-bold font-rale text-center py-4">
+          New Password
+        </h3>
+
+        <div className="relative w-full mb-2">
+          <label htmlFor="password">
+            Password <span className="text-red-500 font-bold">*</span>:{' '}
+          </label>
+          <input
+            type="Password"
+            onChange={(e) => {
+              setpassword(e.target.value);
+            }}
+            id="password"
+            ref={passRef}
+            required
+            onFocus={() => {
+              setpassFocus(true);
+            }}
+            onBlur={() => {
+              setpassFocus(false);
+            }}
+            className={`w-full rounded-md ${
+              password ? '' : ' border-blue-500 '
+            } border-2 focus:outline-none px-2 py-1 font-bold ${
+              password && validPass ? ' border-green-500' : ' '
+            } ${password && !validPass ? 'border-red-500' : ''}`}
+          />
+          <div
+            className={
+              passFocus && password && !validPass ? 'visible' : 'hidden'
+            }
+          >
+            Must have: <br />
+            <ul className="pl-4 font-bold">
+              <li>One uppercase</li>
+              <li>One lowercase</li>
+              <li>One number</li>
+              <li>At least 6 characters</li>
+            </ul>
+          </div>
+        </div>
+        <div className="relative w-full mb-2">
+          <label htmlFor="password2">
+            Confirm password <span className="text-red-500 font-bold">*</span>:
+          </label>
+          <input
+            type="password"
+            id="password2"
+            onChange={(e) => {
+              setpasswordMatch(e.target.value);
+            }}
+            onFocus={(e) => {
+              setmatchFocus(true);
+            }}
+            onBlur={(e) => {
+              setmatchFocus(false);
+            }}
+            required
+            className={`w-full rounded-md ${
+              passwordMatch ? '' : 'border-blue-500'
+            } border-2 focus:outline-none px-2 py-1 font-bold 
+              ${
+                password && passwordMatch && validMatch
+                  ? 'border-green-500'
+                  : ''
+              } ${
+              password && passwordMatch && !validMatch ? 'border-red-500' : ''
+            }
+              `}
+          />
+          <div
+            className={
+              matchFocus && passwordMatch && !validMatch ? 'visible' : 'hidden'
+            }
+          >
+            Must be similar to password <br />
+          </div>
+        </div>
+        <button
+          className="py-1/2 px-2 text-center bg-primary hover:bg-hover transition-all hover:transition-all uppercase text-white font-bold rounded-md"
+          disabled={!validMatch || !validPass ? true : false}
```

Comment:
- You can just simplify it to  disabled = {!validMatch || !validPass}

- Increase the padding on the button a bit